src/SignalR/server/SignalR/test/Microsoft.AspNetCore.SignalR.TrimmingTests/TestTypedClients.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What mechanism is running this app?

Reply:
We have test infrastructure that runs all these `*.TrimmingTests.proj` and `*.NativeAotTests.proj` projects.

They run on the build machines on the "Test" legs of CI.

![image](https://github.com/dotnet/aspnetcore/assets/8291187/b2ea9708-95a6-4b5b-9afe-2a6f98b7240b)

You can learn more about this at https://github.com/dotnet/aspnetcore/blob/main/docs/Trimming.md#validate-trimming-behavior.